### DISCUSSION TITLE
High Availability

### DISCUSSION THREAD

**Discussion by sreekanthedayar** [chaitin/SafeLine#952] [Q&A] [answered]
When we use WAF, there is a chance of Single point of failure if it doesn't have High Availability. DO you plan any such feature? Or is it already available?

当我们使用WAF时，如果没有高可用性，就有可能出现单点故障。您计划有这样的功能吗？或者已经可以使用了吗？

**Comment by safe1ine** [comment, marked as answer]
For users of the Community and Professional versions of SafeLine with high availability requirements, we recommend deploying two instances of SafeLine and implementing dynamic switching through health checks to achieve high availability. 
The Enterprise version of SafeLine offers high availability capabilities but comes at a higher price.

**Reply by sreekanthedayar** [reply]
You mean by having a Load balancer using haproxy or Nginx infront of Safeline?

**Reply by safe1ine** [reply]
yes
